src/Core/Content/Category/Event/SalesChannelEntryPointsEvent.php

```diff
@@ -0,0 +1,52 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Category\Event;
+
+use Shopware\Core\System\SalesChannel\SalesChannelEntity;
+
+class SalesChannelEntryPointsEvent
```

Comment:
hey @runelaenen 
yesterday a new ADR was accepted and merged: 
https://github.com/shopware/platform/commit/dacf9cf492ba602f0966d422d381c36acb7fe79f

> From now on every new event must implement the `Shopware\Core\Framework\Event\ShopwareEvent` interface. If a `Shopware\Core\System\SalesChannel\SalesChannelContext` is also available, the `Shopware\Core\Framework\Event\ShopwareSalesChannelEvent` interface must be implemented instead.
Some existing events were adjusted, which led to some deprecations.

would be nice if you could consider this new convention :blush: :+1: